### DISCUSSION TITLE
WalletConnect,airdrop

### DISCUSSION THREAD

**Discussion by dafaaifiando** [WalletConnect/walletconnect-monorepo#5385] [Ideas]
WalletConnect, I want to express my sincere gratitude for the incredible ease and security you provide in accessing dApps! Your initiative to offer an airdrop not only rewards the community but also enhances your role as an innovative bridge between users and the expansive world of blockchain. This generous gesture will undoubtedly empower more individuals to explore and engage with decentralized applications, unlocking new opportunities for everyone involved and to making the blockchain experience even more accessible!

**Comment by Thavvaj** [comment]
email otp is not working
